k9mail/src/test/java/com/fsck/k9/notification/NotificationDataTest.java

```diff
@@ -108,6 +107,15 @@ public void testRemoveNotificationForMessageWithRecreatingNotification() throws
     }
 
     @Test
+    public void testRemoveDoesntLeakNotificationIds() {
+        for (int i = 1; i <= NotificationData.MAX_NUMBER_OF_STACKED_NOTIFICATIONS+1; i++) {
```

Comment:
There's some white space missing to satisfy the code style. You can avoid this in the future by importing the code style settings and using Android Studio's "Reformat code" action before committing. See https://github.com/k9mail/k-9/wiki/CodeStyle